app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @user_posts = Post.where(author_id: @user.id)
```

Comment:
- [ Required ] I think it would be better if you use the `recent_comments` method in `post.rb` to render `5` most recent comments in the `app/views/posts/index.html.erb` page. Currently, your app renders all the comments on a post and not the five most recent comments. Please see the screenshot below and rectify this.

## Your design
![Screenshot from 2023-09-15 02-41-22](https://github.com/danielochuba/duo-blog/assets/16392046/451999aa-cb51-4c0d-9613-855ae997e6c8)

## Project requirements
![blog_user_all_posts](https://github.com/danielochuba/duo-blog/assets/16392046/376badd3-e3de-4cfa-a323-bee6cf30c6e2)